src/plugins/discover/public/application/main/utils/fetch_documents.ts

```diff
@@ -45,7 +45,7 @@ export const fetchDocuments = (
   const fetch$ = searchSource
     .fetch$({
       abortSignal: abortController.signal,
-      sessionId: searchSessionId,
+      sessionId: isFetchingMore ? undefined : searchSessionId,
```

Comment:
I know we don't have many unit tests for this file currently, but this could be a good one to add since search sessions can be tricky to deal with.

Reply:
@davismcphee Sure, added via https://github.com/elastic/kibana/pull/163784/commits/cad0221016b9963511550418bf125c67820d7615